### PR TITLE
Change: Heading for 1.11 now

### PR DESCRIPTION
**Pull request by glx22**

### CODE REVIEW

File: bin/ai/compat_1.10.nut [hunk 1]
```diff
@@ -4,3 +4,5 @@
  * OpenTTD is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
  * See the GNU General Public License for more details. You should have received a copy of the GNU General Public License along with OpenTTD. If not, see <http://www.gnu.org/licenses/>.
  */
+
+GSLog.Info("1.10 API compatibility in effect.");
```
Comment (frosch123) [inline bin/ai/compat_1.10.nut]: AILog

Comment (glx22) [reply bin/ai/compat_1.10.nut]: oups :)

**Review by frosch123 — APPROVED** [review]

**Comment by James103** [comment]
Looking at the data format for `_openttd_newgrf_version`, it looks like only 4 bits are allocated to the OpenTTD minor version, which means that there's only 16 possible values that could be stored. Does that mean that, if the format of `_openttd_newgrf_version` is not changed in the meantime, OpenTTD 1.15 must be followed by OpenTTD 2.0?

**Comment by LordAro** [comment]
Yup!

**Comment by LordAro** [comment]
Alternatively, we just overflow into the major version, so that 2.0 == 1.16

**Pull request merged**